designsafe/apps/api/notifications/views/api.py

```diff
@@ -51,9 +51,10 @@ def post(self, request, *args, **kwargs):
         body_json = json.loads(request.body)
```

Comment:
This might be wrong, the POST view.
Can you check where are we using this and see if it's correct?